clusterloader2/pkg/measurement/util/phase_latency.go

```diff
@@ -65,6 +65,24 @@ func (o *ObjectTransitionTimes) Get(key, phase string) (time.Time, bool) {
 	return time.Time{}, false
 }
 
+// DeleteKey removes all entries for given key.
+func (o *ObjectTransitionTimes) DeleteKey(key string) {
```

Comment:
I don't think this function is needed - I don't see any case where we should remove statistics of something that we're measuring.

Reply:
Done.